src/RPVoiceChatMod.cs

```diff
@@ -14,5 +14,12 @@ public override void StartPre(ICoreAPI api)
             config = ModConfig.Config;
             new Logger(api);
         }
+
+        public override void Start(ICoreAPI api)
+        {
+            base.Start(api);
```

Comment:
There is no need to call base methods of the mod system, they are designed to be overwritten